jsonld/collection_frame.jsonld

```diff
@@ -4,6 +4,6 @@
     	"http://www.w3.org/ns/ldp.jsonld"
   	],
     "type": "Collection",
```

Comment:
Perhaps we can also address this incorrect type per #440 before this is merged/handled. Otherwise 👍